server/deployment/treetracker-admin-api-mapping.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: getambassador.io/v2
+kind: Mapping
+metadata:
+  name: treetracker-admin-api
+spec:
+  prefix: /api/admin/
+  service: treetracker-admin-api
+  rewrite: /
```

Comment:
just an fyi, this is the default I think, as is exposing port 80, but since you're using port 80 you're good leaving it unspecified.